webpack.config.js

```diff
@@ -4,61 +4,65 @@ const HtmlWebpackPlugin = require('html-webpack-plugin');
 const ExtractTextPlugin = require('extract-text-webpack-plugin');
 
 module.exports = {
-  context: path.resolve(__dirname, 'examples/src'),
-  entry: {
-    app: './app.js',
-  },
-  output: {
-    path: path.resolve(__dirname, 'examples/dist'),
-    filename: '[name].js',
-    publicPath: '/',
-  },
-  devServer: {
-    contentBase: path.resolve(__dirname, 'examples/src'),
-    host: '0.0.0.0',
-    port: 8000,
-  },
-  module: {
-    rules: [
-      {
-        test: /\.js$/,
-        exclude: [/node_modules/],
-        use: [{
-          loader: 'babel-loader',
-          options: { presets: ['react', 'env'] },
-        }],
-      },
-      { 
-        test: /\.less$/,
-        use: ExtractTextPlugin.extract({
-          fallback: 'style-loader',
-          use: ['css-loader', 'less-loader'],
-        })
-      },
-      {   
-        test: /\.html$/,
-        use: [{
-          loader: 'html-loader',
-        }]   
-      }, 
-    ],
-  },
-  resolve: {
-    alias: {
-      'react-images': path.resolve(__dirname, 'src/Lightbox'),
-    }
-  },
-  plugins: [
-    new webpack.optimize.CommonsChunkPlugin({
-      name: 'common',
-      filename: 'common.js',
-      minChunk: 2,
-    }),
-    new HtmlWebpackPlugin({
-      filename: 'index.html',
-      inject: false,
-      template: path.resolve(__dirname, 'examples/src/index.html')
-    }),
-    new ExtractTextPlugin('example.css'),
-  ]
+	context: path.resolve(__dirname, 'examples/src'),
```

Comment:
why so much space?

Reply:
This project uses tabs instead of spaces for indentation 😭, yet some files, like the webpack config, wasn't using spaces or was using a mix of tabs / spaces, and the eslinter was going crazy here, so formatted it as a part of cleanup according to the projects' rules.